### DISCUSSION TITLE
I am Not Able To Login In Vs Code

### DISCUSSION THREAD

**Discussion by KumarDevelopment** [community/community#73669] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Getting This Error You have not yet finished authorizing this extension to use GitHub. Would you like to try a different way? (device code)

**Comment by Naman-B-Parlecha** [comment]
The error message you're seeing suggests that the extension in Visual Studio Code is not fully authorized to use GitHub. To resolve this, you can try the "device code" authorization method. Here's how:

1. Click on "Try a different way (device code)" when prompted.
2. Follow the instructions provided to complete the authorization process. This usually involves visiting a URL and entering a code.

This should help you finish authorizing the extension to use GitHub. If you encounter any issues during this process, please provide more specific details about your setup for further assistance.

**Reply by Naman-B-Parlecha** [reply]
If u find this helpful please mark this as the answer
Thank you

**Reply by MrVineetRaj** [reply]
but not getting the code either

**Reply by Naman-B-Parlecha** [reply]
what error are u getting can u send in a screen shot??

**Reply by MrVineetRaj** [reply]
Solved now
